app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  validates :email, presence: true,
+    format: {with: VALID_EMAIL_REGEX}
+  validates :name, presence: true,
+    length: {
+      minimum: Settings.validations.digit.min_length_6,
+      maximum: Settings.validations.digit.max_length_50
```

Comment:
bỏ cái min max đi, để length thôi, mới dễ tái sử dung
```suggestion
      minimum: Settings.validations.digit.length_6,
      maximum: Settings.validations.digit.length_50
```